### PR TITLE
Improve coding standard by removing magic literals

### PR DESCRIPTION
**Pull request by wilsonngja**

### CODE REVIEW

File: src/main/java/seedu/duke/Storage.java [hunk 1]
```diff
@@ -553,8 +562,8 @@ public String getUnitTypeLabel(String unitTypeString) {
      * Outputs error message if it's the first error and ignore if it's already printed.
      */
     public void skipClientEntries() {
-        errorClientEntriesCount += 1;
-        if (errorClientEntriesCount == 1) {
+        errorClientEntriesCount += INCREMENT_BY_ONE;
```
Comment (ngdeqi) [inline src/main/java/seedu/duke/Storage.java:565]: lolz errorClientEntriesCount++ works

**Review by ngdeqi — COMMENTED** [review]
LGTM

File: src/main/java/seedu/duke/Storage.java [hunk 2]
```diff
@@ -108,6 +108,15 @@ public class Storage {
     private static final int PROPERTY_RENTAL_PRICE_INDEX = 2;
     private static final int PROPERTY_TYPE_INDEX = 3;
 
+    private static final int CLIENT_PARAMETER_INDEX = 0;
```
Comment (OVReader) [inline src/main/java/seedu/duke/Storage.java:111]: Good use of constants to avoid magic numbers.

**Review by OVReader — APPROVED** [review]
Overall LGTM with many magic numbers avoided.

**Pull request merged**